testing/test_configs/invalid_offline_url.yaml

```diff
@@ -0,0 +1,5 @@
+origin: http://127.0.0.1:51596
+issue_tracker: JIRA
+auth: 
+  type: offline
+  offline_url: INVALID_URL
```

Comment:
Nice! 

Could you also add a test for when auth type is offline & `offline_url` is completely missing?

Reply:
sure!

Reply:
This is done.